src/storage/catalog_class.c

```diff
@@ -1684,6 +1684,12 @@ catcls_get_or_value_from_domain (THREAD_ENTRY * thread_p, OR_BUF * buf_p, OR_VAL
   error =
     catcls_get_subset (thread_p, buf_p, vars[ORC_DOMAIN_SETDOMAIN_INDEX].length, &attrs[8],
 		       catcls_get_or_value_from_domain);
+
+  if (vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length)
```

Comment:
1. vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length **> 0**
2. we should use a predefined function to always read the schema string in the same way. (using tp_String.data_readval is not intuitive, and when someone will change the way schema string is stored, it will have to modify in several places).
3. What about else? I am not sure attrs[9].value is initialized.